README.md

```diff
@@ -117,11 +136,48 @@ Spec:
 ...
 ```
 
+**Note**: When running with `StructuredParameters` you can also check that the
```

Comment:
I tried to make the minimal changes necessary to support both classic DRA and structured parameters in the same code base. It would definitely look different if this weren't a goal. In fact, I would probably move all the state of already-prepared claims out of the CRD and instead checkpoint in a file on the node. Is it worth making this separation in this PR into two separate plugins / controllers that get deployed depending on the `withStructuredParameters` flag?

Reply:
It would make the code more realistic for folks who only want to do structured parameters. I think it's worthwhile.

Reply:
This has now been updated to strip out support for classic DRA